packages/ui-components/src/components/windowedlist.ts

```diff
@@ -1161,10 +1161,22 @@ export class WindowedList<
       this.viewModel.scrollOffset = scrollOffset;
       this._scrollUpdateWasRequested = false;
 
+      if (this._viewport.dataset.isScrolling != 'true') {
+        this._viewport.dataset.isScrolling = 'true';
+      }
+
+      if (this._timerToClearScrollStatus) {
+        window.clearTimeout(this._timerToClearScrollStatus);
+      }
+      // TODO: use `scrollend` event instead once supported by Safari
+      this._timerToClearScrollStatus = window.setTimeout(() => {
+        this._viewport.dataset.isScrolling = 'false';
+      }, 500);
```

Comment:
Re failing tests: given that all other suspected pieces of code were eliminated, this looks like a plausible source of flakiness. I will try implementing `scrollend` support which should help as we are testing on chromium.

The tricky bit is that we do not know if `scrollend` is supported. We may need to allow-list it for Chromium and Firefox? But then testing for browsers is also tricky.